docs/tutorials/advanced_topics/hp_tuning_with_optuna.md

```diff
@@ -45,31 +44,22 @@ In general, the datasets provided by GluonTS are objects that consists of three
 
 We can check details of the `dataset.metadata`.
 
+
 ```python
 print(f"Recommended prediction horizon: {dataset.metadata.prediction_length}")
 print(f"Frequency of the time series: {dataset.metadata.freq}")
 ```
 
-To keep the example small and quick to execute, we are only going to use a subset of the dataset.
-
-
-```python
-from itertools import islice
-train_subset = list(islice(dataset.train, 10))
-test_subset = list(islice(dataset.test, 15))
-```
-
-
 This is what the data looks like (first training series, first two weeks of data)
 
+
 ```python
-to_pandas(train_subset[0])[:14 * 24].plot()
+to_pandas(next(iter(dataset.train)))[:14 * 24].plot()
```

Comment:
There is `first` in `toolz` but it just does the same: https://toolz.readthedocs.io/en/latest/_modules/toolz/itertoolz.html#first

Reply:
I think it’s better to avoid the dependency since it’s just the same

Reply:
The dependency is already there!

Reply:
is it? https://github.com/awslabs/gluon-ts/blob/dev/requirements/requirements-docs.txt

Reply:
https://github.com/awslabs/gluon-ts/blob/483bc5a8aba6a4a92adc0b8b356fe1f6bccb6e00/requirements/requirements.txt#L7